ee/agent/internal/informer_stacks.go

```diff
@@ -52,7 +113,7 @@ func NewStackEventHandler(logger sharedlogging.Logger, membershipClient Membersh
 		if err != nil {
 			panic(err)
 		}
-		for _, module := range GetExpectedModules(stack.GetName(), stacks) {
+		for _, module := range modules {
 			if !slices.Contains(modules, module) {
 				return generated.StackStatus_Progressing
 			}
```

Comment:
The modules should be based from inmemories and not on the one "actually attached" to stack CRD

Reply:
It is the case